bindata/allowlist/daemonset/daemonset.yaml

```diff
@@ -0,0 +1,47 @@
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: cni-sysctl-allowlist-ds
+  namespace: openshift-multus
+spec:
+  selector:
+    matchLabels:
+      app: cni-sysctl-allowlist-ds
+  template:
+    metadata:
+      labels:
+        app: cni-sysctl-allowlist-ds
+    spec:
+      containers:
+        - name: kube-multus-additional-cni-plugins
+          restartPolicy: Never
+          image: centos
```

Comment:
Done

Reply:
Why not use {{.MultusImage}} so it could be overridden via environment variable?
https://github.com/openshift/cluster-network-operator/blob/b84ba70532311716bed1482f668f6487151a50d6/manifests/0000_70_cluster-network-operator_03_deployment.yaml#L70-L71

Reply:
Done